.github/workflows/distcheck.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2000 - 2022 Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# SPDX-License-Identifier: curl
+
+name: dist
+
+on:
+  push:
+    branches:
+    - master
+    - '*/ci'
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@v3
+
```

Comment:
we need to prepare some dependencies for **maketgz**:
```
run: |
sudo apt-get update
apt-get install libgsasl-dev libssl-dev
name: 'install prereqs and impacket'
```

Reply:
I don't think these are required. @bagder I could see you install any of those in Azure CI either, right?

Reply:
The more dependencies configure finds, the more "complete" build it makes but I don't think that matters too much when we just verify the dist.